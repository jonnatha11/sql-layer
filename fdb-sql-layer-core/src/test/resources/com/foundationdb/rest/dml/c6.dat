1	George	Boston
2