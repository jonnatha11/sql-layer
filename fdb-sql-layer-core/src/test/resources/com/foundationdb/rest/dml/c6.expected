[{
	"cid": 2,
	"name": null,
	"address": null
}]